src/App.tsx

```diff
@@ -5,16 +5,35 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [moviesFavoriteList, setMoviesFavoriteList] = useState<Movie[]>([]);
+
+  const addMovieToFavorites = (newMovie:Movie) => {
```

Comment:
```suggestion
  const addMovieToFavorites = (newMovie: Movie) => {
```
add space in between